README.md

```diff
@@ -250,6 +250,7 @@ EQ_MAX_HTTP_POST_CONTENT_LENGTH - The maximum http post content length that the
 EQ_MAX_NUM_REPEATS - The maximum number of repeats the system will allow
 EQ_DEVELOPER_LOGGING - Enable developer style logging described here http://structlog.readthedocs.io/en/stable/development.html
 
+EQ_NEW_RELIC_ENABLED - Enable New Relic monitoring
 NEW_RELIC_LICENSE_KEY - Enable new relic monitoring by supplying a New Relic licence key
 NEW_RELIC_APP_NAME - The name to display for the application in New Relic
```

Comment:
We could but what if we wanted to log both pre-prod and prod to the same New Relic account?